packages/truffle-core/lib/commands/develop.js

```diff
@@ -14,102 +15,83 @@ var command = {
     usage: "truffle develop",
     options: []
   },
-  runConsole: function(config, ganacheOptions, done) {
-    var Console = require("../console");
-    var Environment = require("../environment");
+  async runConsole(config, ganacheOptions, done) {
+    const Console = require("../console");
```

Comment:
Typically, capitalized variables are used for classes and constructor functions. Things like:

```js
const MyClass = require("../MyClass.js")
const thing = new MyClass()
```

In this case, the thing you are requiring from `../console` is just an object literal (or an object "instance", some might say). As a result, it should really be:

```js
const consoleCommand = require("../console")
```

But since this is done everywhere in the code, editing this now might make it difficult to read with the rest of the code. I'll leave this up to you, but I just wanted to point out that this is kind of non-standard.

Reply:
hmmm... I can see the argument for the console command

Reply:
Oh, nm. it's not requiring the console cmd, it's the console lib.

Reply:
issue is we sort've treat all these libs you pointed out as potential classes. i think it makes more sense to keep them capitalized for now. it helps differentiate between `truffle-core/lib/console.js` and `truffle-core/lib/commands/console.js`.

Reply:
i see your point tho and feel there's definitely potential for a much larger restructuring

Reply:
I think I would vote that we change them as we see them.